txserver/main.go

```diff
@@ -4,13 +4,16 @@ import (
 	"context"
 	"encoding/json"
 	"log"
+	"os"
+	"time"
 
 	"github.com/streadway/amqp"
 	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
 )
 
 // a global client that can be used across the package
-var mongoClient *mongo.Client
+var client *mongo.Client
```

Comment:
minor: we should rename this as mongoClient. Otherwise, 'client' can mean any client

Reply:
We will not be using any other client interface in txserver other than redis I think(which we can call cache). MongoClient is too hard to type lol.